### ISSUE TITLE
Temporarily allow a statement to disobey the “Type:” filter, while you’re working on it

### ISSUE THREAD

**Issue by zarino** [mysociety/verification-pages#79]
If you filter to show statements of a particular “type” (let’s say “verifiable”), then start editing a statement, as soon as you progress the statement to the next type (in this case, “actionable”), the statement disappears from your screen.

I understand why this happens – because the statement no longer matches the type you have filtered to see. 

But it’s super annoying, because you’re now in the perfect mood to finish that statement off, but you can’t. You have to re-filter to the new “type” that you _think_ the statement will be in.

Basically it means you’re penalised for filtering to a particular “type”, and instead encouraged to filter to “Show all” types, which means you have to manually “Next… Next… Next…” through potentially hundreds of statements, just to find one you can act on and be sure it doesn’t disappear as soon as you start editing it.

Can we maybe allow statements to “stick around” in your current view, while you’re working on them?

Or maybe there’s a simpler way we can avoid this situation entirely? (Like #80?)
